src/Project.ts

```diff
@@ -34,12 +41,29 @@ export default class Project {
     this.stage = stage;
     this.sprites = sprites;
 
-    Object.freeze(sprites); // Prevent adding/removing sprites while project is running
+    this.targets = [
+      stage,
+      ...Object.values(this.sprites as Record<string, Sprite>),
```

Comment:
Why the `as` phrase here? Can the `sprites` parameter just be typed to this within the constructor itself? Does the Leopard *editor* actually interact with Leopard's TypeScript definitions, and would typing `Project` constructor params cause problems there, or nah? (Do any of the other public classes ex. `Sprite` have any typed parameters yet?)